test/unit/util/obfuscate-sql.test.js

```diff
@@ -37,5 +37,13 @@ tap.test('sql obfuscation', (t) => {
     t.end()
   })
 
+  t.test('should handle large JSON inserts', (t) => {
+    const JSONData = '{"data": "' + new Array(8400000).fill('a').join('') + '"}'
+    const result = obfuscate(
```

Comment:
👍 This is a good test. In the current `main` branch this test fails -- with your changes is passes. 

Thinking out loud -- you mentioned 

> I don't think we are actually running this big queries,

Do you think we need a query that's more representative of the queries in your own system?

Reply:
I did some more exploration and it is actually very similar to what we run, only difference is what we have in JSON data, but that I can't share since that content is confidential